src/app/story/directives/podcast.component.ts

```diff
@@ -33,12 +37,53 @@ export class PodcastComponent implements OnDestroy {
 
   story: StoryModel;
   tabSub: Subscription;
-  itunesExplicitDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesRequirementsDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesExplicitDoc = 'https://support.apple.com/en-us/HT202005';
 
-  OPTIONS = ['Explicit', 'Clean'];
+  OPTIONS = ['', 'Explicit', 'Clean'];
 
-  constructor(tab: TabService) {
-    this.tabSub = tab.model.subscribe((s: StoryModel) => this.story = s);
+  constructor(tab: TabService,
+              private cms: CmsService) {
+    this.tabSub = tab.model.subscribe((s: StoryModel) => this.init(s));
+  }
+
+  init(story) {
+    this.story = story;
+    this.getSeriesOverrides();
+  }
+
+  getSeriesOverrides() {
+    this.cms.account.subscribe(account => {
+      if (this.story.isNew && this.story.parent && this.story.parent.isa('series')) {
+        this.defaultDistributionOverrides(new SeriesModel(account, this.story.parent, false));
+      }
+    });
+  }
+
+  defaultDistributionOverrides(series: SeriesModel) {
+    series.loadRelated('distributions').subscribe(distributions => {
+      this.defaultExplicitToSeries(<DistributionModel[]>distributions);
+    });
+  }
+
+  defaultExplicitToSeries(distributions: DistributionModel[]) {
+    let dist = distributions.find(d => d.kind === 'podcast');
+    dist.loadRelated('versionTemplate').subscribe(() => {
+      dist.loadExternal().subscribe(() => {
+        this.story.loadRelated('versions').subscribe(() => {
+          this.story.versions.forEach(v => {
+            // well crap, new story's version doc is null
```

Comment:
So currently, the only place you see `new AudioVersionModel()` is in the `StoryModel`.  And it always inits the new/existing versions with their template (the `vdoc.follow('prx:audio-version-template')` is in there).  So you don't need to add it as a relation.

That said, it would be nice to revisit/clean-up how many of these related are loaded (and maybe make them all load on-demand rather than preloading).  But I'd prefer to knock that out as an independent PR, since it touches so many things.